app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -72,6 +74,14 @@ class MainActivity : BaseActivity() {
                 R.id.menu_feedback -> {
                     true
                 }
+                R.id.menu_logout -> {
+                    val preferenceManager = PreferenceManager()
+                    preferenceManager.sharedPreferences.edit().clear().apply()
```

Comment:
Should this be in the util class? @m-murad

Reply:
Yes

Reply:
@m-murad I should create a clear function in PreferenceManager, right?